content/blog/Genova.md

```diff
@@ -0,0 +1,43 @@
++++
+draft = false
+type = "blog"
+image = "/blog_img/2018-10-23/genova.jpg"
+thumbImage = "/blog_img/2018-10-23/poster-filter.jpg"
+date = "2018-10-23"
+title = "scOrange representing the FAIR principles in bioinformatics."
+weight = 2
+hardLineBreak = true 
+categories = ["conference", "FAIR", "reproducibility"]
+joinLines = false
+author = "Martin Stražar"
+shortExcerpt = "scOrange's away trip to Genova." 
+longExcerpt = "scOrange's away trip to Genova." 
++++
+
+This week, scOrange was featured on the <a
+href="http://www.igst.it/nettab/2018/">Nettab 2018</a> conference on
+implementing the FAIR principles in bioinformatics, hosted by the charming city
+of Genova. The contemporary reproducibility crisis in science in general has
+led to many initiatives to raise standards in data sharing and management.
+<b>F</b>indability, <b>a</b>ccessibility, <b>I</b>nteroperability, and
```

Comment:
You can use standard markdown instead of hard html. Not necessary to change it here, but for future reference. :)